GravatarApp/AppRoot/RootTabView.swift

```diff
@@ -2,12 +2,19 @@ import Gravatar
 import SwiftData
 import SwiftUI
 
+enum RootTabItem: Int {
+    case gravatar = 0
+    case profile
+    case share
+}
```

Comment:
This could be private. Or nested inside `RootTabView`. If it's nested it could be called `Tab`

Reply:
It's not actually private because it's used by the other Views in the same file, but I made it `fileprivate` for now.